docs/modules/ROOT/pages/access-control.adoc

```diff
@@ -202,3 +134,132 @@ TIP: A recommended configuration is to grant both roles to a secure governance c
+
+```javascriot
+// Token's `restricted` functions closed
+await manager.setTargetClosed(myToken.address, true);
+
+// Token's `restricted` functions open
+await manager.setTargetClosed(myToken.address, false);
+```
+
+WARNING: Consider that even if a manager defines permissions for a contract function. These won't be applied if the managed contract instance is not using the xref:api:access.adoc#AccessManageddf-restricted--[`restricted`] modifier or if its manager is a different one.
+
+=== Role Admins and Guardians
+
+An important aspect of the AccessControl contract is that roles aren't granted nor revoked by role members. Instead, it relies on the concept of a role admin for granting and revoking.
+
+In the case of the AccessMananger, the same rule applies and only each role admin is able to call xref:api:access.adoc#AccessManager-grantRole-uint64-address-uint32-[grant] and xref:api:access.doc#AccessManager-revokeRole-uint64-address-[revoke] functions. Note that calling these functions will be subject to the execution delay the executing role admin has.
+
+Additionally, the access manager stores a _guardian_ an extra protection for each role and it is given the ability of canceling operations that have been scheduled by any role member with an execution delay. Consider that a role will have its initial admin and guardian default to the `ADMIN_ROLE` (`0`).
+
+IMPORTANT: Be careful with the `ADMIN_ROLE` members, since it acts as the default admin and guardian for every role, even a misbehaved guardian can cancel operations at discretion, affecting the manager operation.
+
+=== Manager configuration
+
+The manager contract provides a built-in interface for configuring permission settings. These functions can be accessed for those addresses member of the `ADMIN_ROLE`, which is constantly defined as `0` for all managers.
+
+These functions include:
+
+* Add a label to a role using the xref:api:access.doc#AccessManager-labelRole-uint64-string-[`labelRole`] function.
+* Assign the admin and guardian of a role with xref:api:access.doc#AccessManager-setRoleAdmin-uint64-uint64-[`setRoleAdmin`] and xref:api:access.doc#AccessManager-setRoleGuardian-uint64-uint64-[`setRoleGuardian`].
+* Set each role's grant delay via xref:api:access.doc#AccessManager-setGrantDelay-uint64-uint32-[`setGrantDelay`].
+
+As an admin, some actions you can make will require a delay. Similar to each member's execution delay, some admin operations require waiting for execution and should follow the xref:api:access.adoc#AccessManager-schedule-address-bytes-uint48-[`schedule`] and xref:api:access.adoc#AccessManager-execute-address-bytes-[`execute`] workflow.
+
+These delayed functions are settings scoped to a target, so each target will have a different admin delay that can be adjusted by admins with xref:api:access.doc#AccessManager-setTargetAdminDelay-address-uint32-[`setTargetAdminDelay`]. These actions are:
```

Comment:
> These delayed functions are settings scoped to a target

I don't understand that part.

Reply:
I'm trying to imply that because these functions are for configuring the settings of target contracts, each target will have a different admin delay. I rewrote it to:

> More specifically, these delayed functions are those for configuring the settings of a specific target contract. The delay applied to these functions can be adjusted by the manager admins with xref:api:access.doc#AccessManager-setTargetAdminDelay-address-uint32-[`setTargetAdminDelay`].